### ISSUE TITLE
add affected genes for alleles in mgi

### ISSUE THREAD

**Issue by nlwashington** [monarch-initiative/dipper#207]
there's a bunch of additional information for some alleles that affect >1 gene.

these can be found the in relationship tables.  these are needed to augment the current ingest.

for example, 
http://www.informatics.jax.org/allele/MGI:5614047
affect >40 genes.

@mbrush please consider a compact way of showing variants of this type.  we certainly don't want to create genotype labels with >40 variant loci in them.
